webapi/webapi-iap-xwalk-tests/tests.xml

```diff
@@ -3,24 +3,34 @@
 <test_definition>
   <suite category="W3C/HTML5 APIs" name="webapi-iap-xwalk-tests">
     <set name="iap" type="js">
-      <testcase component="Crosswalk APIs/Experimental/InApp Payment API" execution_type="auto" id="Navigator_iap" purpose="Test Navigator iap" subcase="2">
+      <testcase component="Crosswalk APIs/Experimental/InApp Payment API" execution_type="auto" id="Navigator_iap" purpose="Test Navigator iap" subcase="23">
         <description>
-          <test_script_entry>/opt/webapi-iap-xwalk-tests/iap/Navigator_iap.html</test_script_entry>
+          <test_script_entry>/iap/Navigator_iap.html</test_script_entry>
         </description>
       </testcase>
-      <testcase component="Crosswalk APIs/Experimental/InApp Payment API" execution_type="auto" id="InAppPurchase" purpose="Test InAppPurchase interface" subcase="6">
+      <testcase component="Crosswalk APIs/Experimental/InApp Payment API" execution_type="auto" id="Navigator_iap_init" priority="P1" purpose="Test Navigator iap init method" status="approved" type="compliance" subcase="2">
         <description>
-          <test_script_entry>/opt/webapi-iap-xwalk-tests/iap/InAppPurchase.html</test_script_entry>
+          <test_script_entry>/iap/Navigator_iap_init.html</test_script_entry>
```

Comment:
The prefix /opt/xxx is used for the "root" app only. For the sub apk, this prefix is not added automatically as it will be copied to the sub-app directory when packing. Currently, google and xiaomi are two sub apps.